### DISCUSSION TITLE
10 static doesnt serve on production

### DISCUSSION THREAD

**Discussion by DylanPrinsloo** [community/community#127093] [Programming Help] [answered]
### Hi everyone,

Me and my partner seem to have difficulty running one of our static logos on our server which then correlates to the Landing page logo not rendering. Too much detail to explain but anyone have a good idea on how we could approach this? Maybe some guideline to how we can render it on the page?

**Comment by DylanPrinsloo** [comment]
@brandenvs can tell you a bit more on the problem, if you need some information and/or guidance on the situation.

**Comment by brandenvs** [comment, marked as answer]
The issue has been resolved. The problem stemmed from a 403 response. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403

403 responses are challenging to resolve as the cause is vague and ambiguous. In the end, the UNIX permissions on the production server caused the 403. Once the permissions were adjusted appropriately, the STATIC resources were served securely. More specifically, the files within the STATIC directory did not have rwx(read-write-execute), so NGINX was unable to render or serve them.

**Reply by ebndev** [reply]
@brandenvs thanks for updating with the answer!
